components/Navbar/Navbar.html

```diff
@@ -33,21 +34,25 @@
         </button>
         <nav class="menu">
           <ul id="tabs">
-            <li class="tab"><a href="/index.html">Home</a></li>
+            <li class="tab"><a href="/pathsphere/index.html">Home</a></li>
             <li class="tab">
-              <a href="/pages/scholarships/scholarships.html">Scholarships</a>
+              <a href="/pathsphere/pages/scholarship-feature/scholarships.html"
+                >Scholarships</a
+              >
             </li>
-            <li class="tab"><a href="/pages/jobs/jobs.html">Jobs</a></li>
-            <li class="tab"><a href="/pages/forum/forum.html">Forum</a></li>
-            <li class="tab"><a href="/pages/faqs/faqs.html">FAQs</a></li>
-            <li class="tab"><a href="/pages/blog/blog.html">Blog</a></li>
+            <li class="tab"><a href="../../pages/jobs/jobs.html">Jobs</a></li>
+            <li class="tab">
+              <a href="../../pages/forum/forum.html">Forum</a>
+            </li>
+            <li class="tab"><a href="../../pages/faqs/faqs.html">FAQs</a></li>
+            <li class="tab"><a href="../../pages/blog/blog.html">Blog</a></li>
           </ul>
         </nav>
         <div class="cursor"></div>
 
-        <img src="/images/dark.png" id="icon" alt="" />
+        <img src="../../images/dark.png" id="icon" alt="" />
       </nav>
     </header>
-    <script src="/components/Navbar/Navbar.js"></script>
+    <script src="./Navbar.js"></script>
```

Comment:
```suggestion
    <script src="/components/Navbar/Navbar.js"></script>
```

Reply:
@BhaktiMore18 click on commit suggestions on this, it will help you earn a github badge